docs/reference/ml/ml-shared.asciidoc

```diff
@@ -963,10 +964,11 @@ is `shared`, which generates an index named `.ml-anomalies-shared`.
 end::results-index-name[]
 
 tag::results-retention-days[]
-Advanced configuration option. The number of days for which job results are 
-retained. Once per day at 00:30 (server time), results older than this period
-are deleted from {es}. The default value is null, which means results are
-retained.
+Advanced configuration option. When set this option will prune results older
```

Comment:
I'm not sure I understand why we're making the descriptions diverge like this (focus on pruning vs retaining). I'd recommend keeping them in synch unless I'm missing something important here. For example:

```suggestion
Advanced configuration option. The period of time (in days) that results are retained.
```